index.rst

```diff
@@ -44,14 +44,648 @@
+For example, faro could be used to estimate the effective depth of individual images by computing the flux of a point source that would be measured with signal-to-noise of 10, or to measure variations in effective depth across the focal plane that would be indicative of variable atmosphere transparency. 
+The intent is that these metrics would be available within minutes to the observers to inform nighttime operations. 
+faro is designed to be modular and configurable so that additional metrics can be readily added to support summit operations.
+
+faro is NOT itself a visualization tool, but rather generates scalar metric values that could be used as input to visualization tools.
+
+
+Available Computing Power
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. TODO: Robert needs to write this
+
+The following computing resources are available for use but how the hardware will be utilized is not yet well planned and/or documented.
+
+- Camera Diagnostic Cluster
+- Commissioning Cluster (Antu)
+   
+
+The use of these clusters remains unclear and is called out in the `Other Findings and Identified Issues`_ section, under the :ref:`Diagnostic_and_Commissioning_Cluster` heading.
+
+
+.. _Deliverable 4:
+
+Deliverable 4: Augmenting Current and Adding New Functionalities
+----------------------------------------------------------------
+
+.. note:: 
+
+   The deliverable description from the `charge`_ has been directly copied here to ease readability.
+
+   1. A mapping of the use-cases into the currently available systems, clearly identifying where new functionality is required.
+
+      - This could be by augmenting current systems or the creation of a new system if required
+      - Deliver a proposed implementation for each use-case
+
+Each of the use-cases presented in `Deliverable 1`_ contain a heading regarding a suggested implementation.
+The contents of each section refer to new and/or augmented functionality that is seen accross many of them.
+Because the explicit identification of new functionality would add unnecessary noise and confusion for the reader, the content is accumulated here and explained in greater detail.
+
+The items for this deliverable have been separated into two areas:
+
+#. A description of areas where `Entirely New Functionality`_ is required.
+#. A description where the requirements can be met by `Augmenting Current Functionality`_.
+
+This working group also created a proof-of-concept of the critcal implementation recommendations and found them to be successful in satisfying the requirements and being relatively straightforward to implement.
+Details are found in the `Proof-of-concept Demonstrations`_ section.
+
+One should also note that there were functionalities that that group found to be critical to the success of commissioning, but not directly for on-the-fly applications, which therefore resulted in the requirement being out-of-scope. 
```

Comment:
Not sure if there is a duplicated "that" or not... can you give this phrase a second pass?

functionalities that that group -> functionalities that the group